test/gtest/common/test.cc

```diff
@@ -144,8 +144,8 @@ test_base::wrap_errors_logger(const char *file, unsigned line, const char *funct
                               ucs_log_level_t level, const char *prefix,
                               const char *message, va_list ap)
 {
-    /* Ignore warnings about empty memory pool */
-    if (level == UCS_LOG_LEVEL_ERROR) {
+    /* Ignore warnings about empty memory pool or EP failure */
+    if ((level == UCS_LOG_LEVEL_ERROR) || (level == UCS_LOG_LEVEL_WARN)) {
```

Comment:
IMHO would be better to change  ucs_warn("Error %s was not handled for ep %p"...) to ucs_error("Unhandled error %s for ep %p"...)